lib/go-tc/servers.go

```diff
@@ -962,6 +962,10 @@ type ServerV40 struct {
 	CommonServerProperties
 	Interfaces        []ServerInterfaceInfoV40 `json:"interfaces" db:"interfaces"`
 	StatusLastUpdated *time.Time               `json:"statusLastUpdated" db:"status_last_updated"`
+	ConfigUpdateTime  *time.Time               `json:"configUpdateTime" db:"config_update_time"`
+	ConfigApplyTime   *time.Time               `json:"configApplyTime" db:"config_apply_time"`
+	RevalUpdateTime   *time.Time               `json:"revalUpdateTime" db:"revalidate_update_time"`
+	RevalApplyTime    *time.Time               `json:"revalApplyTime" db:"revalidate_apply_time"`
```

Comment:
Default to epoch in the DB now. `COALESCE` removed.

Reply:
I still believe these need to be nullable since they aren't required to be sent to the server in requests from clients. If I remove the pointer, does that mean clients MUST fill a value in here?

Reply:
Clients wouldn't have to fill in the value, but that would cause TO to think it actually meant to send "epoch 0" (the default value it would see after unmarshalling the request body). What you're saying makes sense -- I guess they should be pointers.